### PR TITLE
Measurements API to return alias measurements.

### PR DESCRIPTION
**Pull request by jasebell**

### CODE REVIEW

File: src/clj/kixi/hecuba/api/measurements.clj [hunk 1]
```diff
@@ -34,6 +34,12 @@
 
 (def ^:private uploads-status-resource-path (p/resource-path-string :uploads-status-resource))
 
+(defn get-measurements-keys [sensor-keys session]
```
Comment (Eleonore9) [inline src/clj/kixi/hecuba/api/measurements.clj]: Does `keys` mean `ids` here? I'm not sure of what are measurements keys or sensor keys.

File: src/clj/kixi/hecuba/api/measurements.clj [hunk 2]
```diff
@@ -34,6 +34,12 @@
...
+  (let [sensor (sensors/get-by-type sensor-keys session)]
+    (if-let [alias-sensor (:alias-sensor sensor)]
```
Comment (Eleonore9) [inline src/clj/kixi/hecuba/api/measurements.clj]: I don't get what's a sensor alias. Is it an integer?
So you either return this value or a map with device/sensor info.

Comment (Eleonore9) [inline src/clj/kixi/hecuba/api/measurements.clj]: What happens if a device has several sensors?

Comment (jasebell) [inline src/clj/kixi/hecuba/api/measurements.clj]: You are essentially return a map of keys. The id's are uuid type keys, not integers.

**Comment by Eleonore9** [comment]
Ok! From what I understand you can :shipit:

**Pull request merged**